docs/get-started/testnets-and-devnets.md

```diff
@@ -7,28 +7,18 @@ image: ../img/og/og-getstarted-testnets.png
 --- 
 
 To make the difference between Cardano mainnet functionality and testnet functionality clearer, we moved the old content of [developers.cardano.org](https://developers.cardano.org) to [testnets.cardano.org](https://testnets.cardano.org) with the launch of this developer portal.
-## Cardano testnet
-The [Cardano testnet](https://testnets.cardano.org/en/testnets/cardano/overview/) is your playground when testing [Cardano integration](/docs/integrate-cardano/), building with [transaction metadata](/docs/transaction-metadata/), exploring [native tokens](/docs/native-tokens/) or learning how to [operate a stake pool](/docs/operate-a-stake-pool/).
+## Cardano testnets
+The [Cardano testnets](https://testnets.cardano.org/en/testnets/cardano/overview/) are your playgrounds when testing [Cardano integration](/docs/integrate-cardano/), building with [transaction metadata](/docs/transaction-metadata/), exploring [native tokens](/docs/native-tokens/) or learning how to [operate a stake pool](/docs/operate-a-stake-pool/).
 
-Testnet is an essential part of the development process for Cardano, as it allows developers to test and refine their code before deploying it on the live network, ultimately improving the reliability and security.
-
-### Where to get test ada?
-
-Because the [Cardano testnet](https://docs.cardano.org/cardano-testnet/overview/) is an independent network, separate from the Cardano mainnet, it requires its own token: test ada (tAda).
-
-Test ada is worth nothing. With it you can safely perform all tests free of charge - the reason why you want to develop on the testnet. 
-
-To get free test ada, you need to visit: [Cardano Testnet Faucet.](https://docs.cardano.org/cardano-testnet/tools/faucet)
-
-In depth explanation about Cardano Testnet Faucet can be found [here.](/docs/integrate-cardano/testnet-faucet/)
+The Testnets are an essential part of the development process for Cardano, as they allow developers to test and refine their code before deploying it on the live network, ultimately improving the reliability and security.
 
 ### What testnet should I use?
 
 #### Preview Testnet
 
-- The preview testnet is a testing environment used to showcase new features and functionality to the Cardano community before they are deployed on the mainnet.
+- The Preview Testnet is a testing environment used to showcase new features and functionality to the Cardano community before they are deployed on the mainnet.
```

Comment:
```suggestion
- The Preview Testnet is a testing environment used to showcase new features and functionality to the Cardano community before they are deployed on the Mainnet.
```
This is a big style question (and currently contraindicated by https://developers.cardano.org/docs/portal-style-guide ... @katomm?) but if Testnet X is currently capitalised as a unique proper name, then since "mainnet" is also a unique proper name then it should be capitalised as well 😬 ... not just on this page, but everywhere on the Dev Portal. 😫